buildtest/buildsystem/base.py

```diff
@@ -441,45 +426,49 @@ def run_tests(self, testfile):
 
         # Change to the test directory
         os.chdir(self._get_testdir())
+        self.logger.debug(f"Changing to directory {self._get_testdir()}")
 
         # Run the test file using the shell
         cmd = [self.get_shell(), testfile]
+
+        self.logger.debug(f"Running Test via command: {cmd}")
+
         command = BuildTestCommand(cmd)
         out, err = command.execute()
 
         # Record the ending time
         self.metadata["end_time"] = datetime.datetime.now()
 
         # Keep an output file
-        run_output_file = os.path.join(
-            self.metadata.get("rundir"), "%s.out" % self.build_id
-        )
+        run_output_file = os.path.join(self.metadata.get("rundir"), self.build_id)
 
-        # Run the test file, print output to file
-        with open(run_output_file, "w") as fd:
+        # Keep an error file
```

Comment:
Having run_output_file and run_error_file is redundant, you can remove the second and then add an extension for err or out to the own variable.

Reply:
thats quite odd, i recall making this change but for some reason this was not made something must have happened. Let me fix this quick

Reply:
This is fixed now